### PR TITLE
This adds squanchy to the dictionary 

### PR DESCRIPTION
**Pull request by jakeHooleyARU**
is this ok?

### CODE REVIEW

File: test_people.py [hunk 1]
```diff
@@ -9,6 +9,14 @@ def test_oliver():
     assert oliver.hobbies == []
 
 
+def test_squanchy():
+    squanchy = setup_people()['squanchy_smith']
+    assert squanchy.first_name == 'squanchy'
+    assert squanchy.age == 21
+    assert squanchy.hobbies == ['squanching','running']
```
Comment (osmart) [inline test_people.py]: according to the Git Actions CI/CD pytest.
```
./test_people.py:16:45: E231 missing whitespace after ','
./test_people.py:20:1: E303 too many blank lines (3)
./people.py:29:60: E231 missing whitespace after ','
./people.py:33:1: E302 expected 2 blank lines, found 1
2     E231 missing whitespace after ','
1     E302 expected 2 blank lines, found 1
1     E303 too many blank lines (3)
```
test_squancy should have a space after the comma on line 16 etc.

Please make sure that pycodestyle is clean before commit and check the GitHub actions flake8 log is clean (see video on Canvas).

**Review by osmart — CHANGES_REQUESTED** [review]

File: people.py [hunk 2]
```diff
@@ -22,6 +22,14 @@ def _add_oliver(people):
     key = new_p.first_name + '_' + new_p.surname
     people[key] = new_p
 
+
+def _add_squanchy(people):
+    """ adds squanchy """
+    new_p = PERSON(first_name='squanchy', surname='smith',
+                   age=21, pet='Rat', hobbies=['squanching','running'])
```
Comment (osmart) [inline people.py]: Need a white space after the comma `['squanching','running']`

Comment (osmart) [reply people.py]: Responding to this PR review is TW5 exercise please see:

https://canvas.anglia.ac.uk/courses/14266/discussion_topics/132367

File: test_people.py (hunk 1 above)
Comment (jakeHooleyARU) [reply test_people.py]: fixed in 751fc96

is this ok?

File: people.py (hunk 2 above)
Comment (jakeHooleyARU) [reply people.py]: fixed in f25f16f
is this ok?

**Pull request merged**